sssoja/tv-bland
# So/fix width cast list

### What changes have you made?
- Fixed a small inconsistency on the `header` padding
- Improved responsive padding on mobile 
- Tweaked the column widths on the cast list 

### Which issue(s) does this PR fix?
Fixes #59 

### Screenshots (if there are design changes)
No visible design changes

### How to test
Go to the homepage and the show page and check:
- that there is enough padding on mobile and desktop
- there should still be no text overflow on any of the character names in the cast list
